### PR TITLE
✨ kaltura: add serviceUrl

### PR DESCRIPTION
**Pull request by mrhein**
Implement serviceUrl http://player.kaltura.com/docs/api#kWidget.apiOptions
This is needed to use amp-kaltura with kaltura comunity edition, also this is a must have for TCF v2 https://iabeurope.eu/tcf-2-0/ .
Otherwise if you implement kaltura ce over iframe you can't consent your users.

### CODE REVIEW

**Comment by mrhein** [comment]
@googlebot I signed it!

**Comment by mrhein** [comment]
@dvoytenko it looks like @kristoferbaxter is currently in parental leave.
Could someone else review this part?

File: extensions/amp-kaltura-player/0.1/amp-kaltura-player.js [hunk 1]
```diff
@@ -65,6 +67,9 @@ class AmpKaltura extends AMP.BaseElement {
     setIsMediaComponent(this.element);
 
     this.entryId_ = this.element.getAttribute('data-entryid') || 'default';
+
+    this.serviceUrl_ =
```
Comment (dvoytenko) [inline extensions/amp-kaltura-player/0.1/amp-kaltura-player.js:71]: No restrictions at all on the what the origin could be here?

**Comment by dvoytenko** [comment]
LGTM, just one clarification.

File: extensions/amp-kaltura-player/0.1/amp-kaltura-player.js (hunk 1 above)
Comment (mrhein) [reply extensions/amp-kaltura-player/0.1/amp-kaltura-player.js:71]: Every kaltura CE installation are possible, https://github.com/kaltura/platform-install-packages/releases/tag/Propus-16.5.0 could be done by everyone, it is like amp-iframe.
Do you have security concern's? Another solution would be to have a array with possible publishers and check if it is part of it. But this implied every publisher have to make a pull request and validation of it's setup.

Comment (dvoytenko) [reply extensions/amp-kaltura-player/0.1/amp-kaltura-player.js:71]: Got it.

File: extensions/amp-kaltura-player/OWNERS [hunk 2]
```diff
@@ -4,7 +4,11 @@
 {
   rules: [
     {
-      owners: [{name: 'itaykinnrot'}, {name: 'ampproject/wg-ui-and-a11y'}],
+      owners: [
+        {name: 'itaykinnrot'},
+        {name: 'ampproject/wg-ui-and-a11y'},
+        {name: '@mrhein'},
```
Comment (dvoytenko) [inline extensions/amp-kaltura-player/OWNERS:10]: It looks like @ is not allowed here. Could you pls remove?

**Comment by dvoytenko** [comment]
Looks good. Just need to pass tests now. It looks like `OWNERS` file format complains. I made a comment on that file. PTAL.

**Comment by mrhein** [comment]
@dvoytenko should be fixed now

**Comment by dvoytenko** [comment]
@mrhein PTAL the unit test failure here: https://travis-ci.org/github/ampproject/amphtml/jobs/714893662

File: extensions/amp-kaltura-player/validator-amp-kaltura-player.protoascii [hunk 3]
```diff
@@ -32,6 +32,10 @@ tags: {  # <amp-kaltura-player>
   html_format: AMP
   tag_name: "AMP-KALTURA-PLAYER"
   requires_extension: "amp-kaltura-player"
+  attrs: {
+    name: "data-service-url"
+    mandatory: false
```
Comment (Gregable) [inline extensions/amp-kaltura-player/validator-amp-kaltura-player.protoascii:37]: false is the default, so no need to list it explicitly

File: extensions/amp-kaltura-player/0.1/test/validator-amp-kaltura-player.html [hunk 4]
```diff
@@ -37,6 +37,16 @@
      layout="responsive" width="480" height="270">
   </amp-kaltura-player>
 
+  <!-- valid kaltura ce example -->
+  <amp-kaltura-player
+    data-service-url="front.video.funke.press"
```
Comment (Gregable) [inline extensions/amp-kaltura-player/0.1/test/validator-amp-kaltura-player.html:42]: nit, optional: alignment appears off on this line.

File: extensions/amp-kaltura-player/validator-amp-kaltura-player.protoascii (hunk 3 above)
Comment (mrhein) [reply extensions/amp-kaltura-player/validator-amp-kaltura-player.protoascii]: is removed

File: extensions/amp-kaltura-player/0.1/test/validator-amp-kaltura-player.html (hunk 4 above)
Comment (mrhein) [reply extensions/amp-kaltura-player/0.1/test/validator-amp-kaltura-player.html:42]: fixed

**Comment by mrhein** [comment]
@dvoytenko @Gregable looks good for me.

File: extensions/amp-kaltura-player/OWNERS (hunk 2 above)
Comment (mrhein) [reply extensions/amp-kaltura-player/OWNERS]: removed

**Comment by dvoytenko** [comment]
@mrhein Thanks! Merging.

**Comment by mrhein** [comment]
@dvoytenko Thanks, does this feature has a chance to get in the next release?

**Comment by dvoytenko** [comment]
@rsimha @danielrozenberg could you pls estimate when this merge can make it to release?

**Comment by danielrozenberg** [comment]
If all goes smoothing: to opt-in next Tuesday, to 1% of traffic the day after, and to the remaining 99% of traffic the week after. You can always reference the [release schedule](https://amp.dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule/) doc on amp.dev for this question 🙂

**Pull request merged**